docs/content/connectors/hive.md

```diff
@@ -9,6 +9,10 @@ materialize feature view to a Hive table.
 
 1. Only supports Hadoop 3.1.x
 
+## Supported format
+
+- Flink: text, CSV, Parquet
```

Comment:
What is the definition of the "text" format?

Note that it is not specified in https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/table/formats/overview/,  `formats/README.md`, and `formats/filesystem.md`.

Reply:
The "text" format is mentioned in Hive connector's document[1]. I think Flink's document here[2] might not be strict enough, as Protobuf is not listed in this sheet either.

[1] https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/table/hive/hive_read_write/#formats
[2] https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/table/formats/overview/